feg/gateway/services/aaa/servicers/accounting.go

```diff
@@ -321,6 +327,16 @@ func createSessionOnSessionManager(mac net.HardwareAddr, subscriberId *lte_proto
 	return session_manager.CreateSession(req)
 }
 
+func activateSessionOnSessionManager(subscriberId *lte_protos.SubscriberID) (*lte_protos.UpdateTunnelIdsResponse, error) {
+	req := &lte_protos.UpdateTunnelIdsRequest{
+		Sid:      subscriberId,
+		BearerId: 0,
+		EnbTeid:  0,
+		AgwTeid:  0,
+	}
+	return session_manager.UpdateTunnelIds(req)
```

Comment:
I think this approach (having AAA call UpdateTunnelIds AND CreateSession)  is good, but maybe we can consider renaming `UpdateTunnelIds` to `ActivateSession`?  Can we also add a comment about the 0 values so it's clear these values are garbage values?

Reply:
Yep, on future PRs `UpdateTunnelIds` will be updated to `ActivateSession`. It neest to be updated on MME too